sentry_sdk/integrations/django/asgi.py

```diff
@@ -0,0 +1,30 @@
+"""
+Instrumentation for Django 3.0
+
+Since this file contains `async def` it is conditionally imported depending on
+Django version.
```

Comment:
Does this refer to https://github.com/getsentry/sentry-python/pull/573/commits/adb3fd3d770a1db1e4c62ebc8f3a7e1d41391940#diff-f7309d947f9ef191e39296742d71f72aR322?

If so, that's less so about the Django version and more about the existence of an import path.

Reply:
I am referring to the conditional import of *this* file in `sentry_sdk.integrations.django.__init__`. This file is invalid syntax on Python 2.

Reply:
note: clarify what the actual condition is

Reply:
updated, thanks!